src/main/java/org/symphonyoss/symphony/messageml/elements/TextArea.java

```diff
@@ -0,0 +1,98 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import org.commonmark.node.Node;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.markdown.nodes.form.FormElementNode;
+import org.symphonyoss.symphony.messageml.util.XmlPrintStream;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TextArea extends FormElement {
+
+  public static final String MESSAGEML_TAG = "textarea";
+
+  private static final String PLACEHOLDER_ATTR = "placeholder";
+  private static final String REQUIRED_ATTR = "required";
+  private static final List<String>
+      VALID_VALUES_FOR_REQUIRED_ATTR = Arrays.asList("true", "false");
+
+  private final static String MARKDOWN = "Text Area";
+
+  public TextArea(Element parent) {
+    super(parent, MESSAGEML_TAG);
+  }
+
+  @Override
+  public void validate() throws InvalidInputException {
```

Comment:
Missing validation that content inside tag is only TextNode, see "assertContentModel".